babel.config.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
Otherwise, `nx build core` throws

```
Error during bundle: Error: Babel was run with rootMode:"upward" but a root could not be found when searching upward from "/Users/n.barsukov/WebstormProjects/maskito/projects/core/src".
One of the following config files must be in the directory tree: "babel.config.js, babel.config.cjs, babel.config.mjs, babel.config.json".

```